pra_request_tracker/record_requests/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+
+from pra_request_tracker.record_requests.models import Agency
+from pra_request_tracker.record_requests.models import RecordRequest
+from pra_request_tracker.record_requests.models import RecordRequestFile
+
+
+class AgencyForm(forms.ModelForm):
+    class Meta:
+        model = Agency
+
```

Comment:
Nit: spaces in between class attributes can be removed

Reply:
Sorry, what do you mean by this?

Reply:
Oh never mind, I see it, I was looking at the wrong module 🤦